frontend/packages/knative-plugin/src/components/traffic-splitting/TrafficSplittingFields.tsx

```diff
@@ -39,7 +39,7 @@ const TrafficSplittingFields: React.FC<Props> = ({ revisionItems, values }) => {
       <TrafficModalRevisionsDropdownField
         name="revisionName"
         revisionItems={items}
-        title={t('knative-plugin~Select a revision')}
+        title={t('knative-plugin~Select a Revision')}
```

Comment:
Revision is not a proper noun so it doesn't need to be capitalized.

Reply:
Revision is as well resource created by Serverless

Reply:
Sounds good, thanks for clarifying! No need to change then.